installer/scripts/service_control

```diff
@@ -271,7 +271,7 @@ enable_omsagent_service()
             local omsagent_initd=/etc/init.d/$OMSAGENT_WS
             cp /etc/opt/microsoft/omsagent/sysconf/omsagent.ulinux $omsagent_initd
 
-            sed -i s,WORKSPACE_ID=.*,WORKSPACE_ID=$WORKSPACE_ID,1 $omsagent_initd
+            sed -i s,%WORKSPACE_ID%,$WORKSPACE_ID,g $omsagent_initd
```

Comment:
1 means replace only the first occurrence; g means replace globally.
The old one assume there is one occurrence matches the pattern. Since I change it to use symbol, and add a new place, it should become a global match.